toolchain/check/control_flow.cpp

```diff
@@ -0,0 +1,130 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "toolchain/check/control_flow.h"
```

Comment:
Should this IWYU a bit more?

Reply:
Added some includes that it seemed like I could...